bundles/org.openhab.binding.nikohomecontrol/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -595,7 +599,8 @@
 		<description>@text/channelAlarmArmedDescription</description>
 		<category>Alarm</category>
 		<tags>
-			<tag>Alarm</tag>
```

Comment:
Does not seem related to the `LockState`, was this intentional? 
If so, i doubt this is an improvement, what is your argument to change this?

Reply:
> Does not seem related to the LockState, was this intentional?

Yes.

> what is your argument to change this?

The channel seems to relate to whether the alarm system is armed or not. So it is not actually an Alarm, but rather the setting of the system between armed or disarmed. So I think operation mode is in fact more appropriate.

Reply:
Ah I see